drivers/hmis/app/graphql/types/forms/pick_list_option.rb

```diff
@@ -19,7 +19,26 @@ def self.options_for_type(pick_list_type, user:)
       case pick_list_type
       when 'COC'
         ::HUD.cocs_in_state(ENV['RELEVANT_COC_STATE']).sort.map do |code, name|
-          { code: code, label: name, secondary_label: code }
+          { code: code, label: "#{code} - #{name}" }
+        end
+
+      when 'STATE'
+        states = JSON.parse(File.read('drivers/hmis/lib/pick_list_data/states.json'))
+        states.map do |obj|
+          {
+            code: obj['abbreviation'],
+            label: "#{obj['abbreviation']} - #{obj['name']}",
+            initial_selected: obj['abbreviation'] == ENV['RELEVANT_COC_STATE'],
+          }
+        end
+
+      when 'GEOCODE'
+        geocodes = JSON.parse(File.read("drivers/hmis/lib/pick_list_data/geocodes/geocodes-#{ENV['RELEVANT_COC_STATE']}.json"))
```

Comment:
Not sure we need to reload this each day, but if the process is relatively light, then that feels like a reasonable cache time.
```suggestion
        geocodes = Rails.cache.fetch(['GEOCODES', ENV['RELEVANT_COC_STATE']], expires_in: 1.days) do   
          JSON.parse(File.read("drivers/hmis/lib/pick_list_data/geocodes/geocodes-#{ENV['RELEVANT_COC_STATE']}.json"))
        end
```

Reply:
I missed the map below, but that can go in the cache too.

Reply:
Updated! thanks